labs/coding-101-rest-basics-ga/2.md

```diff
@@ -40,4 +40,6 @@ In this example, you request the list of hosts, and that information is returned
 
 (<a href="http://www.json.org/" target="_blank">JSON</a> -- JavaScript Object Notation, is a lightweight text-based open standard designed for human-readable data interchange.)
 
+(<a href="http://https://www.w3schools.com/xml/xml_whatis.asp/" target="_blank">XML</a> -- Extensible Markup Language, a language designed to store and transport data.)
```

Comment:
There's a double http then https here, plus I think you can delete the end slash /. And usually I see the acronym spelled out with a capital X, like eXtensible.

Holler if you want help with the `git commit --amend` and then `git push`.